code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling.dm

```diff
@@ -0,0 +1,160 @@
+/**
+ * # Spiderlings
+ *
+ * Baby spiders that are generated through a variety of means (like botany for instance).
+ * Able to vent-crawl and eventually grow into a full fledged giant spider.
+ *
+ */
+/mob/living/basic/spiderling
+	name = "spiderling"
+	desc = "It never stays still for long."
+	icon_state = "spiderling"
+	faction = list(FACTION_SPIDER)
+	speed = 1
+	move_resist = INFINITY // YOU CAN'T HANDLE ME LET ME BE FREE LET ME BE FREE LET ME BE FREE
+	speak_emote = list("hisses")
+	basic_mob_flags = FLAMMABLE_MOB
+	mob_size = MOB_SIZE_TINY
+
+	unique_name = TRUE
+	gold_core_spawnable = HOSTILE_SPAWN // because of what we grow into!
+
+	// we have _some_ bite
+	melee_damage_lower = 2
+	melee_damage_upper = 4
+
+	health = 5 // very low.
+	maxHealth = 5
+	unsuitable_cold_damage = 4
+	unsuitable_heat_damage = 4
+	death_message = "lets out a final hiss..."
+
+	ai_controller = /datum/ai_controller/basic_controller/spiderling
+
+	// VERY red, to fit the eyes
+	lighting_cutoff_red = 22
+	lighting_cutoff_green = 5
+	lighting_cutoff_blue = 5
+
+	/// The mob we will grow into.
+	var/mob/living/basic/giant_spider/grow_as = null
+	/// The message that the mother left for our big strong selves.
+	var/directive = ""
+	/// Simple boolean that determines if we should apply the spider antag to the player if they possess this mob. TRUE by default since we're always going to evolve into a spider that will have an antagonistic role.
+	var/apply_spider_antag = TRUE
```

Comment:
We need to think of a less stupid system for this really because currently we give antagonist status to anyone who enters a (non-araneus) spider mob by any means... but it's probably not required for this PR.

Reply:
yeah this PR is already too large and that would have to encompass both giant spiders (out of scope) and spiderlings in a way